src/screens/shared/dashboard/StatCards.js

```diff
@@ -22,49 +35,175 @@ const styles = {
     textAlign: 'center'
   }
 };
-class StatCards extends React.PureComponent {
-  getCardStats = () => {
-    // TODO: Replace with Airtable Call
+class StatCards extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      farms: [],
+      ksFarms: [],
+      totalHarvests: [],
+      recentHarvests: [],
+      gapCertification: [],
+      numFarmReferred: '',
+      numKsGapAccepted: '',
+      percentKsGapCertified: '',
+      percentKsGapApplied: '',
+      percentGapCertified: '',
+      nsevpHarvestFarms: '',
+      totalHarvestsPounds: '',
+      percentGapApplied: ''
+    };
+  }
+
+  async componentDidMount() {
+    const farms = await getAllFarmsForFarmSearch();
+    const gapCertification = await getAllGAPCertificationsForStatCard();
+    const totalHarvests = await getAllTotalHarvestsForStatCard();
+    const recentHarvests = await getAllRecentHarvestLogsForStatCard();
+    const ksFarms = farms.filter(farm => farm.ksAffiliated);
+
+    const numGapCertified = gapCertification.filter(farm => farm.gapCertified); // in gap certification there is no ks affiliated column
```

Comment:
@mylesdomingo What is the best way to get all KS-affiliated farms that are GAP certified?
The KS-affiliated field is in the Farms table, but the GAP Certified field is in its own table.